client/src/pages/OfferHelp.js

```diff
@@ -130,19 +131,23 @@ const Step2 = (props) => {
     </WizardStep>
   );
 };
-const WizardButton = styled(SubmitButton)`
-  width: 70%;
-  margin-top: 5rem;
-  margin-left:8rem;
-`;
 const Step3 = (props) => {
   const [email, setEmail] = useState("");
-  const onChange = (evt) => {
-    setEmail(evt);
+  const [valid, setValid] = useState(true);
+
+  useEffect(() => {
+    const validated = !email || validateEmail(email);
+    setValid(validated);
+  }, [email]);
+
+  const onChange = (event) => {
+    setEmail(event.target.value);
   };
+
   const onSubmit = () => {
-    props.update("email", email);
+    console.log("submit");
```

Comment:
ah yeah, now i see this one is using `props.update()` that one is passed from the page component below... we could still use that too but I'm not sure how this should work when the wizards is completed

Reply:
We can just have it removed for now and can fix it when the we have the full picture ready.

Reply:
yeah talked on slack about it and it's not clear what has to be done here, good for now